src/main/resources/application.properties

```diff
@@ -1 +1,29 @@
+spring.application.name=cms
+server.servlet.context-path=/cms
 
+##DATABSE
+spring.datasource.url=jdbc:h2:mem:cms_db
+spring.datasource.driverClassName=org.h2.Driver
+spring.datasource.username=poliscrypts
+spring.datasource.password=poliscrypts*2022
+spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
+spring.h2.console.enabled=true
+
+##FLYWAY MIGRATION
+flyway.url=jdbc:h2:mem:cms_db
+flyway.user=poliscrypts
+flyway.password=poliscrypts*2022
+spring.flyway.enabled=true
+
+#JWT
+jwt.prefix=Bearer 
+jwt.expiration=864000000
+jwt.audience=poliscrypts_cms_api
+jwt.iss=cms_api
+#secret key generated based on node crypto lib
+jwt.secretKey=c03829fb6b6b6458cf5b8e65abb0f6628a49753d9096d99c5fd97951bb589895cda64e3ad9172492aae02bccfa2dcb88d4dc6613dc621eeeec530b024f4d76f5
+
+logging.level.org.springframework.web.filter.CommonsRequestLoggingFilter=DEBUG
+server.error.path=/error
+spring.mvc.pathmatch.matching-strategy = ANT_PATH_MATCHER
+spring.cache.jcache.config=classpath:ehcache.xml
```

Comment:
i never use it, but we add functionalities  as needed and recommended :)

Reply:
_**26**_ -> used to enable logging interceptor for http request 
_**27**_ -> to be used in errorController `@RequestMapping("${server.error.path}")`
_**28**_ -> needed for `documentationPluginsBootstrapper` bean of swagger-ui
_**29**_ -> ehache conf location attribue for token blacklisting